test/test_vmap.py

```diff
@@ -3590,7 +3575,7 @@ def test_feature_dropout(self, device, randomness):
             vmap_result = vmap(op, randomness=randomness)(passed)
             torch.manual_seed(seed)
             if randomness == 'different':
-                expected = op(passed.flatten(0, 1))
+                expected = op(passed.flatten(0, 1)) # (B0, B, ...) -> (B0 * B, ...)
```

Comment:
flake8 alert (not sure if it's this line, just needed to put the comment somewhere)